src/code/z_camera.c

```diff
@@ -1536,9 +1536,9 @@ s32 Camera_CalcAtForFriendlyLockOn(Camera* camera, VecGeo* eyeAtDir, Vec3f* targ
             deltaY = focalActorPosRot->pos.y - *yPosOffset;
             temp_f0_6 = Math_FAtan2F(deltaY, OLib_Vec3fDistXZ(at, &camera->eye));
 
-            if (temp_f0_6 > 0.34906584f) { // (M_PI / 9)
+            if (temp_f0_6 > 0.34906584f) { // (F_PI / 9)
```

Comment:
Huh, I wonder if `(f32)(M_PI / 9)` (the double) would match here, probably not tho

Reply:
It actually does :)